src/hoi4_world/countries/hoi4_country_converter.cpp

```diff
@@ -174,6 +174,40 @@ std::vector<hoi4::EquipmentVariant> DetermineActiveVariants(const std::vector<ho
    return active_variants;
 }
 
+
+std::set<std::string> ConvertLaws(const std::set<std::string>& vic3_laws, std::string_view ideology)
+{
+   std::set<std::string> hoi4_laws;
+
+   // civilian economy with export focus is the default
+   hoi4_laws.emplace("civilian_economy");
+   hoi4_laws.emplace("export_focus");
+
```

Comment:
vicky3 free trade should map to hoi4 free trade

Reply:
While thematically appropriate, free trade is very uncommon in vanilla HoI4 (USA and rebels + China), and adding this rule makes at fairly common (about 10% of countries).